WordPress/src/jetpack/res/values/styles_toolbar.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Minor (🔍): This `xmlns:android` namespace is not necessary for now. Consider removing it.

PS: Same apply for the `main` such file.

Reply:
In 0cc8acd, 3cbf8c4